test/integration/features/user_creates_from_url.spec.js

```diff
@@ -20,13 +18,6 @@ describe('authenticated e2e-user', function() {
   let namespaceForFixtures = "template-dumpster";
 
   let setupEnv = function() {
```

Comment:
Want to leave a comment here as well?  I imagine we will forget about `extensions/test-extensions.js` in the future since its not really a real feature anymore.